crates/moonbuild/src/entry.rs

```diff
@@ -448,25 +448,21 @@ fn convert_moonc_test_info(
 
 #[allow(clippy::too_many_arguments)]
 pub fn run_test(
-    moonc_opt: &MooncOpt,
-    moonbuild_opt: &MoonbuildOpt,
+    moonc_opt: MooncOpt,
+    moonbuild_opt: MoonbuildOpt,
```

Comment:
why change the param type from &MoonbuildOpt to MoonbuildOpt and use Arc::clone() below?

Reply:
`MoonbuildOpt`(and `MooncOpt` and `ModuleDB`) need to be shared in multiple threads, so `Arc` is needed. And the `Arc::new` want to move the value instead of getting a reference, so I change the parameter.